app/src/main/java/live/hms/app2/ui/meeting/pinnedvideo/VideoListAdapter.kt

```diff
@@ -180,6 +187,21 @@ class VideoListAdapter(
               holder.binding.nameInitials.text = NameUtils.getInitials(payload.name)
               holder.binding.name.text = payload.name
             }
+            is PeerUpdatePayloads.NetworkQualityChanged -> {
+              context?.let { context ->
```

Comment:
Use this context instead storing a reference.
`holder.binding.root.context`

Reply:
ohh